lib/features/pokemon_overview/pokemon_overview_vm.dart

```diff
@@ -14,5 +14,5 @@ class PokemonOverviewVmFactory extends VmFactory<AppState, PokemonOverviewConnec
 class PokemonOverviewVm extends Vm {
   /// TODO: add other additional fields here
   PokemonOverviewVm({required this.pokemonList}) : super(equals: [pokemonList]);
-  final List<Pokemon> pokemonList;
+  final List<PokemonDto> pokemonList;
```

Comment:
see feedback above then use the typedef for your type here and also rename this variable to `pokemons` 🙂

Reply:
@yohancai any update here? 🙂